finance-tracker/app/controllers/user_stocks_controller.rb

```diff
@@ -1,2 +1,20 @@
 class UserStocksController < ApplicationController
+  def create
+    stock = Stock.find_by(ticker: (params[:ticker]))
+    if stock.blank?
+      stock = Stock.new_lookup(params[:ticker])
+      stock.save
+    end
+    stock.users << current_user
+    flash[:notice] = "Stock #{stock.name} was successfully added to your portfolio"
+    redirect_to portfolio_index_path
+  end
+
+  def destroy
+    stock = Stock.find(params[:id])
+    user_stock = UserStock.where(user_id: current_user.id, stock_id: stock.id).first
+    user_stock.destroy
```

Comment:
```suggestion
    Stock.find(params[:id]). user_stocks.destroy_by(user_id: current_user.id)
```
I think it can be done the above. Please verify

Reply:
yep it works. but I'll do 

```ruby
stock = Stock.find(params[:id])
stock.user_stocks.destroy_by(user_id: current_user.id)
```

because I need the stock variable below for 
` flash[:notice] = "Stock #{stock.ticker} was successfully removed from your portfolio"`